sql-plugin/src/main/scala/com/nvidia/spark/rapids/NvtxWithMetrics.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2019,2021, NVIDIA CORPORATION.
```

Comment:
This file was changed in 2020, copyright should have been updated.
```suggestion
 * Copyright (c) 2019-2021 NVIDIA CORPORATION.
```